livre/eq/Node.cpp

```diff
@@ -129,6 +119,17 @@ class Node
         _textureDataCachePtr.reset();
     }
 
+    void updateAndSendFrameRange()
```

Comment:
Could just be called updateFrameRange. The fact that the app's frame range is updated via event is an implementation detail.